docs/UserGuide.md

```diff
@@ -74,19 +74,32 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding an internship entry: `add`
 
-Adds a person to the address book.
+Adds a new internship entry to the list of existing entries.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/COMPANY_NAME r/ROLE s/STATUS d/DATE`
```

Comment:
Might want to add the particular format that we are using for date
Eg. YYYY-MM-DD or YYYY-DD-MM

Reply:
Yup good point! Should I include it in this section, or should it be stated in the _Notes about the command format_ section since there are multiple commands that involve dates?

Reply:
In that case should we put status on top as well?

Reply:
Hmmm... I think can put on top to prevent repetition? What do the others think? @potty10 @DerrickSaltFish @kohkaixun

Reply:
We can put it at the top.